.eslintrc.js

```diff
@@ -55,7 +55,7 @@ module.exports = {
     // auto-fixable: Respect all Prettier rules and apply it.
     "prettier/prettier": "error",
     // not-auto-fixable: No unused variables allowed.
-    "no-unused-vars": "error",
+    "no-unused-vars": "off",
```

Comment:
Why is this rule turned off? It's supposed to raise an error.

Reply:
I used it because for some states I was using setState value and not the original value like const [note,setNote] = useState(false) so for note it was showing error as I was only using setNote. I would go through the code again to use an alternative to remove the error.

Reply:
>as I was only using setNote

@nancy-binani Can you please point out the real world use case of declaring a React state and not having any `state` value but only the `setter`?

> used it because for some states I was using setState value and not the original value

Also, I can't find such a state declaration in this PR. Can you please share the link to the lines of code from this PR where I can see that change? Thanks.

Reply:
Actually I was using the state but commented it because it wasn't required now . It was for addNote so I removed commented line from this PR and commited this code and I forgot to remove onClick function as well.  So for this PR this commit was giving error. I am really sorry for the confusion. Would recheck every code again and do the commits and PR very carefully from now onwards.

Reply:
Ok got it. That's fine.